### PR TITLE
New package: CubicSDR-0.2.5

### PR DESCRIPTION
**Pull request by bra1nwave**

### CODE REVIEW

**Comment by newbluemoon** [comment]
Probably needs libatomic-devel and most likely also `-latomic` for i686 and armv6l.

**Comment by maxice8** [comment]
ping

**Comment by bra1nwave** [comment]
Updated to 0.2.5 and added the proposed fixes (Thanks again, newbluemoon).

File: srcpkgs/liquid-dsp/template [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Template file for 'liquid-dsp'
```
Comment (maxice8) [inline srcpkgs/liquid-dsp/template]: This package doesn't have a versioned shared library

Comment (bra1nwave) [reply srcpkgs/liquid-dsp/template]: I added patch from Debian to fix this.

**Pull request closed without merging**